Plugin~/WebRTCPlugin/GraphicsDevice/Vulkan/VulkanGraphicsDevice.cpp

```diff
@@ -19,6 +19,10 @@ namespace unity
 {
 namespace webrtc
 {
+#if !VULKAN_USE_CRS
+    static VulkanGraphicsDevice* s_GraphicsDevice = nullptr;
+#endif
```

Comment:
I think this can go as well actually.
This was for the AccessQueueCallback, but with removing that we can get rid of this as well I believe.